extensions/web/02_event.js

```diff
@@ -1117,6 +1117,10 @@
   }
 
   class MessageEvent extends Event {
+    get source() {
+      return null;
+    }
```

Comment:
Added to appease WPT.

Reply:
Interesting... points to this module being in need of a rewrite :wink:.